lib/list.js

```diff
@@ -0,0 +1,21 @@
+import { getLanguage } from './common.js';
+
+const nbsp = '\xa0';
+
+/**
+ * Returns a region-specific list separator for the current language.
+ * @param { Boolean } nonBreaking Whether to replace spaces with non-breaking spaces.
+ */
+export function getSeparator({ nonBreaking } = {}) {
+	const langTag = getLanguage();
+	const space = nonBreaking ? nbsp : ' ';
+
+	// In this function, most languages return the separator used by the Intl.ListFormat() API with options:
+	//		{ style: 'short', type: 'conjunction' }
+	// 		https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat
+	// Arabic, due to some inconsistencies, returns the separator used by Google Translate.
+
+	if (langTag === 'ar') return `${space}،${space}`;
+	if (['ja', 'zh-cn', 'zh-tw'].includes(langTag)) return '、';
```

Comment:
I was wondering why we didn't put a `${space}` here but I realized that there's already a space in the character `、`, for anyone else wondering.

Reply:
Yeah, sort of - that `、` is one single wide character, with its own built-in space. Doesn't seem to use an added space with it.

Reply:
Yeah that's what I meant haha. Visually it looks as though a separate space was put after the comma, but it's actually just whitespace embedded within the character.